compiler-core/src/language_server/tests/action.rs

```diff
@@ -10554,3 +10555,119 @@ pub fn main() {
         find_position_of("let c").select_until(find_position_of("* d"))
     );
 }
+
+#[test]
+fn annotate_all_top_level_definitions_function() {
+    assert_code_action!(
+        ANNOTATE_TOP_LEVEL_DEFINITIONS,
+        r#"
+pub fn add_two(thing) {
+  thing + 2
+}
+
+pub fn add_one(thing) {
+  thing + 1
+}
+"#,
+        find_position_of("fn").select_until(find_position_of("("))
+    );
+}
+
+#[test]
+fn annotate_all_top_level_definitions_constant() {
+    assert_code_action!(
+        ANNOTATE_TOP_LEVEL_DEFINITIONS,
+        r#"
+pub const answer = 42
+
+pub fn add_two(thing) {
+  thing + 2
+}
+
+pub fn add_one(thing) {
+  thing + 1
+}
+"#,
+        find_position_of("const").select_until(find_position_of("="))
+    );
+}
+
+#[test]
+fn annotate_all_top_level_definitions_already_annotated() {
+    assert_no_code_actions!(
+        ANNOTATE_TOP_LEVEL_DEFINITIONS,
+        r#"
+pub const answer: Int = 42
+
+pub fn add_two(thing: Int) -> {
```

Comment:
This test has a syntax error, so the action will not appear no matter what!
```suggestion
pub fn add_two(thing: Int) -> Int {
```